ts/components/conversation/Linkify.tsx

```diff
@@ -22,9 +23,25 @@ export class Linkify extends React.Component<Props> {
   };
 
   public render() {
-    const { text, renderNonLink } = this.props;
-    const matchData = linkify.match(text) || [];
+    const { text, renderNonLink, isRss } = this.props;
     const results: Array<any> = [];
+
+    if (isRss && text.indexOf('</') !== -1) {
+      results.push(
+        <div
+          dangerouslySetInnerHTML={{
+            __html: text
+              .replace(/(<? *script)/gi, '')
```

Comment:
Do we want to use a library or something to sanitise the text instead of us manually removing script tags?
I feel this isn't very adequate and might miss some very small cases which could become a problem.

Reply:
yes, this is the main reason for the WIP

Reply:
chose jquery's script filter. Looked at using an NPM but couldn't figure out how to deal with the yarn/freeze thing we have going on.